Functions/Gherkin.ps1

```diff
@@ -787,3 +787,150 @@ function ConvertTo-HashTableArray {
     }
 }
 
+function Import-FeatureFile {
+    param(
+        # Path to Search
+        [Parameter(ValueFromPipeline = $true)]
+        [string[]]
+        $Path = $pwd
+    )
+    
+    begin {
+        $parser = New-Object Gherkin.Parser
+    }
+
+    process {
+        $item = Get-Item -Path $Path
+
+        if ($item.psisContainer) {
+            foreach ($file in $path | Get-ChildItem -Recurse -Filter *.feature) {
+                try {
+                    $fixtureFilename = "{0}.Steps.ps1" -f $file.BaseName
+                    $fullName = Join-Path -Path $file.Directory -ChildPath $fixtureFilename
+                    $parsed = $parser.parse($file.FullName)
+
+                    if (Test-Path -Path $fullName) {
+                        Update-TestStubs -stubFile $fullName -parsedInput $parsed
+                    }
+                    else {
+                        $stubFile = New-Item -Path $fullName -ItemType File
+                        
+                        Add-Stubs -stubFile $stubFile -parsedInput $parsed   
+                    }
+                }
+                catch {
+                    throw $_
+                }
+            }   
+        }
+        else {
+            try {
+                $fixtureFilename = "{0}.Steps.ps1" -f $item.BaseName
+                $fullName = Join-Path -Path $item.Directory -ChildPath $fixtureFilename
+                $parsed = $parser.parse($item.FullName)
+
+                if (Test-Path -Path $fullName) {
+                    Update-TestStubs -stubFile $fullName -parsedInput $parsed
+                }
+                else {
+                    $stubFile = New-Item -Path $item.Directory -Name $fixtureFilename -ItemType File
+                    
+                    Add-Stubs -stubFile $stubFile -parsedInput $parsed
+                    
+                }
+            }
+            catch {
+                throw $_
+            }
+        }
+    }
+
+    end {
+
+    }
+
+
+}
+
+function Add-Stubs {
+    param(
+        # File to write stubs into
+        [Parameter(Mandatory = $true)]
+        [string]
+        $stubFile,
+        # Parsed Gherkin Data
+        [Parameter(Mandatory = $true)]
+        [Gherkin.Ast.GherkinDocument]
+        $parsedInput
+    )
+    
+    foreach ($scenario in $parsed.feature.children) {
+
+        $string = ""
+
+        foreach ($step in $scenario.steps) {
+            $string += New-GherkinStub -step $step
+        }
+
+        Add-Content -Value $string -Path $stubFile
+    }
+}
+
+function New-GherkinStub {
+    param(
+        # The Gherkin Step to Stub
+        [Parameter(Mandatory = $true)]
+        [Gherkin.Ast.Step]
+        $step
+    )
+    
+    $outString = $step.keyword + "`""
+    
+    $description = $step.text
+    
+    $regmatch = [regex]::new("<\w*>").matches($description)
```

Comment:
I don't understand the assumption you're making here about having angle brackets in your step text...

Reply:
The documentation I'v seen so far says that if you have variables in the step text that correspond to something like an examples table, that the word representing the variable name should be enclosed in brackets. [Cucumber Documentation](https://github.com/cucumber/cucumber/wiki/Scenario-Outlines)
That regex just seemed an easy way to detect when someone was attempting that and to figure out what the variable name should be. If that's not the case I'm happy to find some other way to detect that.

Reply:
Oh, yes, `Scenario Outlines` ... that's right. Of course, it's not quite simple as that, there are also doc strings and data tables, and once people start customizing the [step definition](https://cucumber.io/docs/reference#step-definitions), anything can have parameters... which is why you need to use `Find-GherkinStep`

Reply:
I'll definitely need to do that. I discovered not long after this PR that my parameterization code wasn't working properly in cases of things like doc strings or passing in JSON strings and such.